doc/source/tutorials/bipartite_matching/assets/bipartite_matching.py

```diff
@@ -0,0 +1,42 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+
+# Assign nodes 0-4 to one side, and the nodes 5-8 to the other side
+g = ig.Graph.Bipartite(
+    [0, 0, 0, 0, 0, 1, 1, 1, 1],
+    [(0, 5), (1, 6), (1, 7), (2, 5), (2, 8), (3, 6), (4, 5), (4, 6)]
+)
+assert(g.is_bipartite())
```

Comment:
It is a bad practice to use parentheses around an `assert` call because assert is a _statement_ in Python, not a function. `assert(x)` happens to be okay, but `assert(x, failure_message)` is parsed as `assert (x, failure_message)`, where `(x, failure_message)` is treated as a tuple of length 2, which is always true. In other words:

```python
>>> assert False, "this assertion is False"    # expected to fail
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: this assertion is False
>>> assert(False, "this assertion is False")   # will not fail because it is interpreted as a tuple
```

TL;DR: please remove the parentheses